ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationDelegate.swift

```diff
@@ -117,12 +119,20 @@ import UIKit
                     vc.finishedCallback = finishedCallback
                 }
                 vc.navigateWithRoute = self.viewController?.navigateWithRoute
+                vc.globalProperties = self.viewController?.globalProperties
                 self.viewController?.navigationController?.pushViewController(vc, animated: true)
             }
         }
         return completion("success")
     }
 
+    private func combineRouteData(routeData: [AnyHashable: Any], globalProperties: [AnyHashable: Any]?) -> [AnyHashable: Any] {
+        guard let globalProps = globalProperties else {
+            return routeData
+        }
+        return routeData.merging(globalProps) {(current, _) in current }
+    }
```

Comment:
@deepueg how do you merge these two dictionaries? if these dictionaries have the same key, which value do you keep? I'm using the one from routeData for now.